app/resources/css-default.template

```diff
@@ -1133,3 +1139,18 @@ table.api-keys td.empty {
     padding: 5px;
   }
 }
+
+.card {
+  background: #ffffff;
+  border: 1px solid #dadada;
+  box-shadow: 2px 2px 3px 0px #AAA;
```

Comment:
Be consistent with the format of the color code. It should be "#a0a0a0", I guess (lowercased 6 digits).